microprofile/WFLY-14281_Integrate-MP-Fault-Tolerance-3.0.adoc

```diff
@@ -0,0 +1,86 @@
+= Integrate MP Fault Tolerance 3.0 into EAP
+:author:            Jason Lee
+:email:             jasondlee@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+MicroProfile 4.0 provides a major upgrade for https://github.com/eclipse/microprofile-fault-tolerance/releases/tag/3.0[MicroProfile Fault Tolerance 3.0] with changes described in its
+https://download.eclipse.org/microprofile/microprofile-fault-tolerance-3.0/microprofile-fault-tolerance-spec-3.0.html#release_notes_30[Release Notes].
+
+This component upgrade also covers the upgrade of SmallRye Fault Tolerance that implements MicroProfile Fault Tolerance 3.0.
+
```

Comment:
I think that here in the overview it should be noted here the fact that the major upgrade will introduce breaking changes in the spec itself and also in the integration with the Metrics implementation which is provided in the same runtime context (i.e. WildFly). 
What do you think about mentioning it here and move the "Breaking changes" section here for it to be a subsection of the "Overview"?

Reply:
Yeah, moved from bottom to here.